### DISCUSSION TITLE
How to deal with Apps from the MS Store

### DISCUSSION THREAD

**Discussion by PauliBaumeister** [svrooij/WinTuner#143] [Q&A] [answered]
Hi, i don't know how to deal with apps that are listed from the Microsoft Store, e.g. Clipchamp (a low end video editing tool from Microsoft). This is the result:
```
PS C:\winget> winget search clipchamp
Name                Id           Version Source
-------------------------------------------------
Microsoft Clipchamp 9P1J8S7CCWWT Unknown msstore
```

I can install Clipchamp with winget and the PacketID:
```
winget install --id 9P1J8S7CCWWT
```

However, when i try to create a package with Wintuner, it fails:
```
PS C:\winget> New-WtWingetPackage -PackageId "9P1J8S7CCWWT" -PackageFolder "C:\winget"
INFO: [WingetRepository] Loading package index from https://raw.githubusercontent.com/svrooij/winget-pkgs-index/main/index.json
INFO: [WingetRepository] Saving package index to cache at C:\Users\TimoHilbertz\AppData\Local\WingetCommunityRepo\index.json
WARNING: Package  not found

```
I think i made the same observation with other apps coming from the Microsoft Store.
Is this an expected behaviour or something that needs to be fixed?

Many thanks for your help!

**Comment by svrooij** [comment, marked as answer]
Have you seen this command? https://wintuner.app/docs/wintuner-powershell/Deploy-WtMsStoreApp

Store apps are not packaged.

**Reply by PauliBaumeister** [reply]
Thank you so much Stephan! And no, i did not realize that there was a dedicated command for this - thanks for pushing me to the right direction!
